dashboard/db/schema.rb

```diff
@@ -508,17 +508,23 @@
   add_index "workshop_attendance", ["segment_id"], name: "index_workshop_attendance_on_segment_id", using: :btree
   add_index "workshop_attendance", ["teacher_id"], name: "index_workshop_attendance_on_teacher_id", using: :btree
 
+  create_table "workshop_cohorts", force: true do |t|
+    t.integer  "workshop_id"
+    t.integer  "cohort_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
   create_table "workshops", force: true do |t|
     t.string   "name"
     t.string   "program_type", null: false
     t.string   "location"
     t.string   "instructions"
-    t.integer  "cohort_id",    null: false
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "time_zone"
```

Comment:
I don't see this in the migrations, does your db have changes not reflected in migrations from other work in progress?